contracts/src/instances/ezeth/ezETHBase.sol

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
This filename needs to be changed. I see several others like this too. Anywhere that we have `ezETHBase` needs to be updated to `EzETHBase`. You might have to use `git mv` for this since MacOS has a case-insensitive file-system.

Reply:
Yeah I'm running into that issue with the other renames right now...